app/views/support_users/dashboard/dashboard.html.slim

```diff
@@ -0,0 +1,3 @@
+- content_for :page_title_prefix, "Hello support user"
+
+<p>Hello support user</p>
```

Comment:
this is a slim file

Reply:
ah but I guess this is just temporary. cool

Reply:
Fair point, I don't remember which of us added that.  Probably me.  Not sure I can be bothered to change it since it's being replaced anyway and still works as intended.